extras/module-node/Makefile

```diff
@@ -14,3 +14,4 @@ test:
 	./test 'var ape = require("ape"); ape.module.filename === "ape.js" ? "module.filename OK" : "module.filename FAILED";'
 	./test 'var ape = require("ape"); ape.module.loaded === true && ape.wasLoaded === false ? "module.loaded OK" : "module.loaded FAILED";'
 	./test 'var ape = require("ape"); ape.__filename === "ape.js" ? "__filename OK" : "__filename FAILED";'
+	./test 'var badger = require("badger"); badger.foo === 123 && badger.bar === 234;'
```

Comment:
The test is trying to check that just assigning `exports.foo = <something>` works because none of the other tests do that: they all replace `module.exports` entirely.